python/grass/temporal/temporal_topology_dataset_connector.py

```diff
@@ -56,6 +54,8 @@ class TemporalTopologyDatasetConnector:
 
             start = start.next()
 
+    .. code-block:: python
+
```

Comment:
Adding a separate block here might be right. One block is plain code, the other is with prompts and results.

Reply:
The blackdoc didn't run for this because all comments under ```.. code-block:: python``` must have >>> before them. This is why I moved it before >>> import.

Reply:
The documentation says it supports rst and here are code blocks without `>>>` which are formatted by blackdoc in the example: https://blackdoc.readthedocs.io/en/latest/usage.html

Reply:
Should I move ```.. code-block:: python``` as it was? I just need to add >>> before the comments.  I moved the```.. code-block:: python```  before  >>>import because I got an error like below.
```
error: cannot format C:\msys64\usr\src\grass\python\grass\temporal\temporal_topology_dataset_connector.py: Invalid indentation: 59:  >>> import grass.temporal as tgis

Oh no! 💥 💔 💥
1 file failed to reformat.
```

It becomes like this.
```
    .. code-block:: python
        >>> # We have build the temporal topology and we know the first map
        start = first
        while start:
            >>> # Print all maps this map temporally contains
            dlist = start.get_contains()
            for map in dlist:
                map.print_info()

            start = start.next()
```

Reply:
Looks like a problem with blackdoc. We are not officially using blackdoc (it not in the CI, pre-commit, or guidelines), so let's not force blackdoc-specific issues on the code.

Let's make it a separate block (so two `.. code-block:: python` instead of the original one) and let's get back to blackdoc later.

Reply:
Please let me know if I misunderstood two code blocks. First, I added the code for the comment "# We have built..." and second, I added the code for >>>Import.
```
    .. code-block:: python

        # We have build the temporal topology and we know the first map
        start = first
        while start:

            # Print all maps this map temporally contains
            dlist = start.get_contains()
            for map in dlist:
                map.print_info()

            start = start.next()

    .. code-block:: python
         >>> import grass.temporal as tgis
```

Reply:
This is correct. Two blocks is what I meant.